sdk/servicebus/service-bus/src/serializers/subscriptionResourceSerializer.ts

```diff
@@ -55,6 +61,7 @@ export function buildSubscriptionOptions(
     UserMetadata: getStringOrUndefined(subscription.userMetadata),
     ForwardDeadLetteredMessagesTo: getStringOrUndefined(subscription.forwardDeadLetteredMessagesTo),
     AutoDeleteOnIdle: getStringOrUndefined(subscription.autoDeleteOnIdle),
+    // TODO: EntityAvailabilityStatus does not exist in .net
```

Comment:
EntityAvailabilityStatus was present in WindowsAzure.ServiceBus package i.e. what we call the T0
Both .Net and Java T1 packages did not have it and the service team does not have an explanation for it.
For T2, they recommended to keep it, but also mentioned that it is read-only. So, am guessing user cannot set it but can read it?

Reply:
CC: @JoshLove-msft, who might want to add it. 

@ramya-rao-a , I'll remove the comment since we're...already correct :)

Reply:
Removed the comment from JS (field is staying, obviously).